index.html

```diff
@@ -35,23 +39,232 @@ <h1>I’m Giorgi Bokuchava glad to see you!</h1>
+        </div>
+        <div class="alltext">
+            <ul>
+                <li class="text1">html</li>
+                <li class="text2">css</li>
+                <li class="text3">javaScript</li>
+            </ul>
+        </div>
+        <button class="but">See Project</button>
+    </section>
+
+
+
+
+
+    <section class="3-page">
+        <div class="snapshot2">
+            <img src="Pictures/snapshot2.png" alt="">
+            <h6>Multi-Post
+                Stories</h6>
+        </div>
+        <div class="bullets2">
+            <h3>CANOPY</h3>
+            <ul class="bullets1">
+                <li>Back End Dev</li>
+                <li>2015</li>
+            </ul>
+        </div>
+        <div class="second-para">
+            <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+        </div>
+        <div class="alltext">
+            <ul>
+                <li class="text1">html</li>
+                <li class="text2">css</li>
+                <li class="text3">javaScript</li>
+            </ul>
+        </div>
+        <button class="but">See Project</button>
+    </section>
+
+
+
+
+
+
+
+    <section class="4-page">
+        <div class="snapshot2">
+            <img src="Pictures/snapshot3.png" alt="">
+            <h6>Tonic</h6>
+        </div>
+        <div class="bullets2">
+            <h3>CANOPY</h3>
+            <ul class="bullets1">
+                <li>Back End Dev</li>
+                <li>2015</li>
+            </ul>
+        </div>
+        <div class="second-para">
+            <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+        </div>
+        <div class="alltext">
+            <ul>
+                <li class="text1">html</li>
+                <li class="text2">css</li>
+                <li class="text3">javaScript</li>
+            </ul>
+        </div>
+        <button class="but">See Project</button>
+    </section>
+
+
+
+
+
+    <section class="5-page">
+        <div class="snapshot2">
+            <img src="Pictures/snapshot4.png" alt="">
+            <h6>Multi-Post
+                Stories</h6>
+        </div>
+        <div class="bullets2">
+            <h3>CANOPY</h3>
+            <ul class="bullets1">
+                <li>Back End Dev</li>
+                <li>2015</li>
+            </ul>
+        </div>
+        <div class="second-para">
+            <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+        </div>
+
+        <div class="alltext">
+            <ul>
+                <li class="text1">html</li>
+                <li class="text2">css</li>
+                <li class="text3">javaScript</li>
+            </ul>
+        </div>
+        <button class="but">See Project</button>
+    </section>
+
+
+
+
+
+
+
+    <section class="aboutme">
+        <div class="headline">
+            <div class="headline-heading">
+                <h1>About myself</h1>
```

Comment:
[OPTIONAL]
 - [ ] Only one `h1` should suppose to be on the page, Kindly make sure you take that into consideration.